package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "electron-forge",
-  "version": "6.0.1",
+  "//": "See lerna.json for version number",
```

Comment:
IMO, I think we can remove this comment or add a comment in the CONTRIBUTING instead of in the package.json (not a merge blocker though)

Reply:
[2da0840](https://github.com/electron/forge/pull/3082/commits/2da084059baafacd8657adeb4c5951b35a93a701)